documentation/src/main/antora/modules/ROOT/pages/_partials/generated_configuration-types.adoc

```diff
@@ -47,3 +47,19 @@ Mark a model (complex object) as being a dataset discovery configuration.
 ----
 
 
+= Checkpoint
+
+Mark a model (complex object) as being a checkpoint configuration and state.
+
+- API: @org.talend.sdk.component.api.input.checkpoint.Checkpoint
+- Sample:
+
+[source,js]
+----
+{
+  "tcomp::configurationtype::name":"test",
```

Comment:
maybe set something else that 'test' ?

Reply:
It's generated automatically, no hand on that.